api/BarProp.json

```
@@ -96,6 +96,54 @@
             "deprecated": false
           }
         }
+      },
+      "visible_popup": {
```

Two changes needed here:

- This should be a subfeature of visible (that is, it should be nested one step deeper, as a sibling of the `__compat` above)
- Let's rename this to "returns_popup" (it's a specified as a getter, so I feel OK using this terminology)

```suggestion
      "returns_popup": {
```